cluster-autoscaler/core/scaleup/orchestrator/orchestrator_test.go

```diff
@@ -696,11 +696,11 @@ func TestScaleUpUnhealthy(t *testing.T) {
 
 	nodes := []*apiv1.Node{n1, n2}
 	nodeInfos, _ := nodeinfosprovider.NewDefaultTemplateNodeInfoProvider(nil, false).Process(&context, nodes, []*appsv1.DaemonSet{}, nil, now)
-	clusterState := clusterstate.NewClusterStateRegistry(provider, clusterstate.ClusterStateRegistryConfig{}, context.LogRecorder, NewBackoff())
+	processors := NewTestProcessors(&context)
+	clusterState := clusterstate.NewClusterStateRegistry(provider, clusterstate.ClusterStateRegistryConfig{}, context.LogRecorder, NewBackoff(), clusterstate.NewDefaultMaxNodeProvisionTimeProvider(&context, processors.NodeGroupConfigProcessor))
```

Comment:
nit: Testing would perhaps be simpler with a test/static maxnodeProvisionTimeProvider - no need to pass `context` or `NodeGroupConfigProcessor`.